trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    return @nameable.correct_name.strip if @nameable.correct_name.length > 10
+
+    @nameable.correct_name
+  end
```

Comment:
- [ ] @alvaroiporre Great work on your decoration, however, the TrimmerDecorator class doesn't work, below is the result from your main.rb file testing the decoration you can see the name doesn't get trimmed to only 10 characters
![image](https://github.com/alvaroiporre/School-Library-Backend/assets/61408860/f5921c3d-6db5-427b-8c35-be18a5859844)

Reply:
Thank you so much for your feedback @danmainah. I can't understand how this decorator must be work. Can you please give me an example to fix the trimmerDecorator.